cmd/sebak/cmd/node.go

```diff
@@ -88,6 +89,8 @@ var (
 	flagTLSCertFile         string = sebakcommon.GetENVValue("SEBAK_TLS_CERT", "sebak.crt")
 	flagTLSKeyFile          string = sebakcommon.GetENVValue("SEBAK_TLS_KEY", "sebak.key")
 	flagValidators          FlagValidators
+	flagSignThreshold       string = sebakcommon.GetENVValue("SEBAK_SIGN_THRESHOLD", "30")
```

Comment:
The current threshold, `30` was only for testing. Please suggest the proper value.

Reply:
60?

Reply:
@outsideris 60% will be meaningful. The default threshold of stellar is 67%, https://github.com/stellar/stellar-core/blob/master/docs/stellar-core_example.cfg#L393